### PR TITLE
docs: Update to Texture Docu - bandit, elevator, logo

### LINKED ISSUE #503: Add docs for bandit mask and extra door / cargo / nose gear decals.
Need documentation for work in PR https://github.com/flybywiresim/a32nx/pull/6959 needs to be added to https://docs.flybywiresim.com/dev-corner/texture-changes/

Proposed text:

## Bandit / Racoon mask.
The bandit/racoon mask is disabled by default.
To enable the mask, in any color of your choice, use the `DECALS_BANDIT_WINDOW_ALBD.PNG.DDS` texture and fill the associated square referenced in the [UV map legend](https://user-images.githubusercontent.com/433429/160307120-ce74ee62-8cb9-4c03-9d37-796a411b4863.png). (Note that the UV legend is 1024x1024, for readability, while the real texture only is 64x64)

## Elevator trim scale
The fill and outline color or the elevator trim scale, is configured in the `DECALS_BANDIT_WINDOW_ALBD.PNG.DDS` texture.
See [UV map legend](https://user-images.githubusercontent.com/433429/160307120-ce74ee62-8cb9-4c03-9d37-796a411b4863.png) for reference. (Note that the UV legend is 1024x1024, for readability, while the real texture only is 64x64)

### LINKED ISSUE #512: Add documentation for logo / sticker / alliance mesh decal
Need documentation for work in PR https://github.com/flybywiresim/a32nx/pull/6982 to be added to https://docs.flybywiresim.com/dev-corner/texture-changes/

Proposed text:

## Logo / sticker / alliance decal
We have added a mesh between the fwd doors and the cockpit, to ease with high quality logos, stickers, etc.
The texture for logo / stickers / alliance is transparent by default.
To enable this texture, add a 2048x2048 `DECAL_LOGO_FRONT_COMP.PNG.DDS` texture and add your graphics in the associated square referenced in the image below.

### Reference
decal-ref
<p align="right"><i>Size and location of usable mesh</i></p>

### PR DESCRIPTION
**Pull request by frankkopp**
Closes #503 
Closes #512

## Summary
Updated texture documentation

### Location
https://docs-git-fork-frankkopp-updt-livery-guide-flybywire.vercel.app/dev-corner/texture-changes/#bandit-racoon-mask

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by frankkopp** [comment]
@bouveng ready to review :)

File: docs/dev-corner/texture-changes.md [hunk 1]
```diff
@@ -45,4 +45,26 @@ If you are the author of a livery that needs individual texturing of these hinge
...
+
+## Bandit / Racoon mask.
+The bandit/racoon mask is disabled by default.
+To enable the mask, in any color of your choice, use the `DECALS_BANDIT_WINDOW_ALBD.PNG.DDS` texture and fill the 
+associated square referenced in the [UV map legend](assets/textures/uv-map-legend.png).
+
+Note that the UV legend is 1024x1024, for readability, while the real texture only is 64x64.
+
+## Elevator trim scale
+The fill and outline color or the elevator trim scale, is configured in the `DECALS_BANDIT_WINDOW_ALBD.PNG.DDS` texture.
```
Comment (Valastiri) [inline docs/dev-corner/texture-changes.md]:
```suggestion
The fill and outline color of the elevator trim scale, is configured in the `DECALS_BANDIT_WINDOW_ALBD.PNG.DDS` texture.
```

File: docs/dev-corner/texture-changes.md [hunk 2]
```diff
@@ -45,4 +45,26 @@ If you are the author of a livery that needs individual texturing of these hinge
...
+See the [UV map legend](assets/textures/uv-map-legend.png) for reference. 
+
+Note that the UV legend is 1024x1024, for readability, while the real texture only is 64x64.
+
+## Logo / sticker / alliance decal
+We have added a mesh between the fwd doors and the cockpit, to ease with high quality logos, stickers, etc.
```
Comment (Valastiri) [inline docs/dev-corner/texture-changes.md]:
```suggestion
We have added a mesh between the forward doors and the cockpit, to ease with high quality logos, stickers, etc.
```
I don't understand the "to ease" portion ... does it mean "to easily integrate high quality [...]"?

**Review by Valastiri — CHANGES_REQUESTED** [review]
Typo's and a question

**Pull request merged**